doc/api/crypto.md

```diff
@@ -2800,6 +2800,47 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### `crypto.randomInt([min, ]max[, callback])`
+<!-- YAML
+added:
+  - REPLACEME
```

Comment:
```suggestion
added: REPLACEME
```

Reply:
@jasnell The previous entry was perfectly fine – I assume this was intentional, we do follow this format for cases in which semver-minor PRs get backported so that we can list multiple versions